manage/installing/installation_minimal_buildout.rst

```diff
@@ -22,6 +22,7 @@ Create a virtual python environment (`virtualenv <https://virtualenv.pypa.io/en/
 
    virtualenv-2.7 zinstance
    cd zinstance
+   source bin/activate
```

Comment:
since it uses explicit the ``bin/pip`` afterwards it does not need any activation. So, it does not hurt.